abtesting/ABTestingExample/Shared/AppConfig.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Copyright 2021 Google LLC
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import Firebase
+
+class AppConfig: ObservableObject {
+  @Published var colorScheme : ColorScheme
+  init() {
+    let value = RemoteConfig.remoteConfig()["color_scheme"].stringValue ?? "nil"
+    colorScheme = value == "dark" ? .dark : .light
+#if DEBUG
+    NotificationCenter.default.addObserver(self,
+                                           selector: #selector(printInstallationAuthToken),
+                                           name: .InstallationIDDidChange,
+                                           object: nil)
+#endif
+  }
+  deinit {
+#if DEBUG
+    NotificationCenter.default.removeObserver(self)
+#endif
+  }
```

Comment:
Since `deinit`'s implementation is only running when `DEBUG`, I think it makes more sense to wrap `deinit` in `#if DEBUG`

Reply:
Done ✅